public/assets/harvard.js

```diff
@@ -143,9 +143,30 @@ $(function() {
   }
 });
 function responsive_search(){
-  $keyword = $("select.search-keyword").children().first();
+	
+  $keyword = $("select.search-keyword").children("option:selected")
+  if ($keyword == null) {
+	  $keyword = $("select.search-keyword").children().first();
+  }
   $down_caret = $("#down-caret");
   $down_caret.appendTo($keyword);
+  $("select.search-keyword").change(function(){
+	  $("select.search-keyword").children().remove($down_caret)
+	  $keyword = $("select.search-keyword").children("option:selected");
+	  $down_caret.appendTo($keyword);
+  });
+  
+  $limit = $("select.limit-field").children("option:selected")
+  if ($limit == null) {
+	  $limit = $("select.limit-field").children().first();
+  }
+  $limit_down_caret = $down_caret.clone();
+  $limit_down_caret.appendTo($limit);
+  $("select.limit-field").change(function(){
+	  $("select.limit-field").children().remove($limit_down_caret)
+	  $limit = $("select.limit-field").children("option:selected");
+	  $limit_down_caret.appendTo($limit);
+  });
```

Comment:
Spoke to @ives1227 about this part. We may be able to refactor and use the same code for both "Limit" and "Search Keyword" carat setup/switching since they're so similar, but there may also be things preventing that from being possible. Looks good to me otherwise!